### ISSUE TITLE
pip install openrocketdoc doesn't work on pip version 8.0.2

### ISSUE THREAD

**Issue by sanketdiwale** [open-aerospace/openrocketdoc#1]
Hi all,

I tried installing openrocketdoc using:
sudo pip install openrocketdoc

This doesn't work on pip version 8.0.2 on my mac OS X computer. 
It returns "No matching distribution found for openrocketdoc".

pip search openrocketdoc also returns no results.

It seems pip cannot find openrocketdoc by default. Is there some source list that needs to be added to find openrocketdoc by pip.

I was able to install the module using "python setup.py install" however.

Best,
Sanket

**Comment by natronics** [comment]
That's correct, I actually haven't published it to pypi yet. This project is still very unstable. Any document created with it now will almost certainly be broken by a future commit.

I also haven't told anyone about this yet, I guess google knows about it though :)

But, since you found it, I'm curious about different use cases that people would be interested in. My original idea was to make modeling historical launch vehicles like the [Aerobee 150](https://open-aerospace.github.io/Aerobee-150/) easier, to be able to compare different hobby simulator outputs (OpenRocket, RASAERO, RockSim, etc.) by converting one from another, and to be able to rapidly create a parametric rocket in code, then run it in a simulator by outputting a valid OpenRocket document for example.

Unfortunately there is a lot yet to do. I wish I had more time!

I'll leave this open until it gets published to pypi. And for anyone else who comes across this issue, for now follow the "Install from source" instructions in the documentation:

https://open-aerospace.github.io/openrocketdoc/installation.html
